setup1-contributor/src/commands/contribute.rs

```diff
@@ -934,11 +935,29 @@ fn update_progress_bar(updater: StatusUpdater, progress_bar: ProgressBar) -> tok
 /// Utility structure to provide updates about the round progress
 struct StatusUpdater {
     server_url: Url,
+    private_key: PrivateKey<Testnet2Parameters>,
     participant_id: String,
 }
 
 impl StatusUpdater {
     async fn status_updater(&self, progress_bar: ProgressBar) -> Result<()> {
+        let status = get_contributor_status(&self.server_url, &self.private_key).await?;
+        match status {
+            ContributorStatus::Queue => {
+                progress_bar.set_message(&"In queue for one of the future rounds");
```

Comment:
"You are in the queue for an upcoming round of the ceremony. Please wait for the prior round to finish, and please stay connected for the duration of your contribution."